### ISSUE TITLE
Ecoregion metric

### ISSUE THREAD

**Issue by lauren-herbine** [Riverscapes/riverscapes-tools#236]
What EcoRegion metric do we want for GNAT and Mississippi reach typing?
What units?
Input data sources

**Comment by joewheaton** [comment]
So right now the Level 4 Eco Regions still are not working in the symbology of Riverscapes Context. That is step 1. After that, I think we need to simply look at those polygons as potential break points for reach types. AS far as for reach typing workflow goes, we need to better understand how consistent (or not) the Ecoregion naming is and/or if there are attributes in there (e.g. descriptions of landscapes as mountains, uplands, lowlands, valleys, etc.) we can use for some high level discrimination.

**Comment by philipbaileynar** [comment]
What does level 4 give us over level 3? Is there some citation or justification that the added detail of level 4 warrants the effort?

**Comment by joewheaton** [comment]
> What does level 4 give us over level 3? Is there some citation or justification that the added detail of level 4 warrants the effort?

Absolutely. We have consistently used Level 4 ecoregions as Landscape Units as our first cuts in Reach Typing (@lauren-herbine) 
[Level 4](https://www.epa.gov/eco-research/level-iii-and-iv-ecoregions-continental-united-states) gives (we only have Level 1, 2 and 3 right now) give us more polygons than Level 3 (see below), which better discriminate specific landscape units as a basis for initial reach breaks. @lauren-herbine you should be thinking about this as a filtering step in your tree (i.e. there are some river styles that only can occur in specific level 4 ecoregions, and many more that occur in multiple potential ecoregions, but not all).  

[Level 1 and 2](https://www.epa.gov/eco-research/ecoregions-north-america) and 3 basically share the same geometry but have more classes in Level 2.  Level 1 has 15 classes, but Level 2 is 50 classes (same exact polygon geometry) and Level 3 picks up to  182 ecoregions. To the untrained eye, they all look exactly the same at scale of HUC 8, but actually have specific symbologies (@lauren-herbine perhaps we should add labels to these?)

## Level 3
![](https://www.epa.gov/sites/production/files/styles/large/public/2015-11/eco_level_iii_us_sm.gif)
We do have this now.

## Level 4
Level 4 is where we get the real detail with 1000s of ecoregions that get very specific (perfect for reach typing):
![](https://www.epa.gov/sites/production/files/styles/large/public/2015-11/eco_level_iv_us_sm.gif). 

For example, in [Warner Lakes]() we go from 2 classes to 5 classes:
![image](https://user-images.githubusercontent.com/9723905/118677667-a29d9080-b7b9-11eb-85f7-f48d91e6b3cf.png)

If we zoom in, this is what it buys us in one catchment (@nick4rivers you and Gus might find this useful too)
![image](https://user-images.githubusercontent.com/9723905/118678663-71719000-b7ba-11eb-89c5-933cff26a2a6.png)


The whole country is only 69 MB ([US Level IV Ecoregions shapefile with state boundaries (69 mb)](https://gaftp.epa.gov/EPADataCommons/ORD/Ecoregions/us/us_eco_l4_state_boundaries.zip)) and the [symbology exists](https://gaftp.epa.gov/EPADataCommons/ORD/Ecoregions/us/Eco_Level_IV_US.lyr). So effort is minimal for getting this going in Arc. I admit that doing for QGIS will be some more work with all these categories. But it is doable.
